src/foam/dao/Relationship.js

```diff
@@ -449,10 +459,15 @@ return junction`
         }, x);
 
         controller.sub('select', function(s, _, id) {
-          dao.find(id).then(function(obj) { self.add(obj); });
+          dao.find(id).then(function(obj) {
```

Comment:
Original seems better. Don't break up to multiple lines if it fits in one reasonable size line.

Reply:
It was greater than 80 chars (which was one of our rules). But yea I'll change it back.